scripts/gen_eval_files.lua

```diff
@@ -213,17 +213,20 @@ end
 
 --- Convert vimdoc references to markdown literals
 --- Convert vimdoc codeblocks to markdown codeblocks
+---
+--- Ensure code blocks have one empty line before the start fence and after the closing fence.
```

Comment:
is this comment still relevant?

Reply:
Yes. Note this is only for the Markdown code blocks that are autogenerated in the `_meta/` files (e.g. `_meta/api.lua`). It ensures that Markdown code blocks have the form:

````
  --- Lorem ipsum etc. etc.
  ---
  --- Example:
> ---                 < -- blank line here!
  --- ```
  --- ...
  --- ...
  --- ```
> ---                 < -- blank line here!
  ---@param foo string Foo
  ---@return table
````

This is not strictly required by the spec, but in my own experience I've found that omitting these blank lines causes issues with some renderers in some cases (including GitHub and `gen_vimdoc.py`).